app/javascript/components/pages/ApartmentIndex.js

```diff
@@ -5,7 +5,14 @@ export default class ApartmentIndex extends Component {
   render() {
     return (
       <>
-      <h3>ApartmentIndex</h3>
+      <h3>Apartments Available</h3>
+        <ul>
```

Comment:
Can you render this list dynamically?  Hint: You will want to iterate over the apartments.